pkgs/desktops/gnome/core/gnome-remote-desktop/default.nix

```diff
@@ -73,7 +85,6 @@ stdenv.mkDerivation rec {
     "-Dsystemd_system_unit_dir=${placeholder "out"}/lib/systemd/system"
     "-Dsystemd_sysusers_dir=${placeholder "out"}/lib/sysusers.d"
     "-Dsystemd_tmpfiles_dir=${placeholder "out"}/lib/tmpfiles.d"
-    "-Dtests=false" # Too deep of a rabbit hole.
```

Comment:
This just re-enables building them. To actually run them, you need `doCheck = true`.

Reply:
Oh, good thing you mentioned this; the build was showing a whole bunch of tests running. Are different tests enabled by removing `-Dtests=false` and setting `doCheck = true;`?

Reply:
Not sure what you have seen – no tests will run unless `doCheck` is `true`.